sympy/printing/latex.py

```diff
@@ -2199,14 +2199,18 @@ def translate(s):
         return s
 
 
-def latex(expr, **settings):
+def latex(expr, fold_frac_powers=False, fold_func_brackets=False, \
+    fold_short_frac=None, inv_trig_style="abbreviated", \
+    itex=False, ln_notation=False, long_frac_ratio=None, \
+    mat_delim="[", mat_str=None, mode="plain", mul_symbol=None, \
+    order=None, symbol_names={}):
     r"""Convert the given expression to LaTeX string representation.
 
     Parameters
     ==========
-    fold_frac_powers : boolean, optional
+    fold_frac_powers : boolean, optional (default = False)
```

Comment:
The defaults are shown in the new function signature that you created, so adding the default here is a redundant.